api/src/test/resources/org/openmrs/module/fhir2/api/dao/impl/FhirObservationDaoImplTest_initial_data_suppl.xml

```diff
@@ -61,4 +61,6 @@
     <obs obs_id="22" person_id="7" concept_id="5092" encounter_id="3" obs_datetime="2008-07-01 00:00:00.0" location_id="1" value_numeric="98.3" comments="" creator="1" date_created="2008-08-18 14:09:35.0" voided="false" value_coded="[NULL]" value_coded_name_id="[NULL]" value_complex="[NULL]" value_text="[NULL]" value_datetime="[NULL]" value_drug="[NULL]" uuid="4efa62d2-6b8b-4803-a8fa-3f32ee54db4f"/>
     <obs obs_id="23" person_id="7" obs_group_id="22" concept_id="5242" encounter_id="3" obs_datetime="2008-07-01 00:00:00.0" location_id="1" value_numeric="14.0" comments="" creator="1" date_created="2008-08-18 14:09:35.0" voided="false" value_coded="[NULL]" value_coded_name_id="[NULL]" value_complex="[NULL]" value_text="[NULL]" value_datetime="[NULL]" value_drug="[NULL]" uuid="744b91f8-bdbc-4950-833b-002244e9fa2b"/>
     <obs obs_id="24" person_id="7" concept_id="5242" encounter_id="3" obs_datetime="2008-07-01 00:00:00.0" location_id="1" value_numeric="134.0" comments="" creator="1" date_created="2008-08-18 14:09:35.0" voided="false" value_coded="5242" value_coded_name_id="1439" value_complex="cbdft" value_text="AFH56" value_datetime="2008-08-18 14:09:35.0" value_drug="[NULL]" uuid="785li1f8-bdbc-4950-833b-002244e9fa2b"/>
-</dataset>
+    <obs obs_id="25" person_id="7" concept_id="5242" encounter_id="3" obs_datetime="2008-07-01 00:00:00.0" location_id="1" value_numeric="100.001" comments="" creator="1" date_created="2008-08-18 14:09:35.0" voided="false" value_coded="[NULL]" value_coded_name_id="[NULL]" value_complex="[NULL]" value_text="[NULL]" value_datetime="[NULL]" value_drug="[NULL]" uuid="89fg071-1f7d-4394-a316-0a458edf28c3"/>
+    <obs obs_id="26" person_id="7" concept_id="5242" encounter_id="3" obs_datetime="2008-07-01 00:00:00.0" location_id="1" value_numeric="0.007" comments="" creator="1" date_created="2008-08-18 14:09:35.0" voided="false" value_coded="[NULL]" value_coded_name_id="[NULL]" value_complex="[NULL]" value_text="[NULL]" value_datetime="[NULL]" value_drug="[NULL]" uuid="56htgf-1f7d-4394-a316-0a458edf28c3"/>
+</dataset>
```

Comment:
Please leave a new line at the end of the file

Reply:
I'll do that in the next commit